Styles building heights

odinkomnogim opened the pull request:
- Grouped styles for heights AND classification
<img width="942" height="578" alt="image" src="https://github.com/user-attachments/assets/91774d53-611d-4993-8f3a-45359741b943" />

- Styles for open data downloading (lines and polygons):
<img width="1733" height="560" alt="image" src="https://github.com/user-attachments/assets/8a199e63-a2bc-4ffb-b53b-64a5a86b28cc" />

Godnik commented:
I added a new style rule if the field 'class_id' is not present and 'building_height' is. Worth checking on the "Demo project"  examples.

odinkomnogim commented:
> I added a new style rule if the field 'class_id' is not present and 'building_height' is. Worth checking on the "Demo project" examples.

Didn't work properly, but I understood what you were saying: we should apply building_heights style (without classification) when 'class_id' is not in attributes, but 'building_height' is. Fixed it
